docs/getting_started/Datus_BI.zh.md

```
@@ -0,0 +1,193 @@
+# BI Dashboard → Subagent 使用指南
+
+*将 BI Dashboard 转换为可复用的 gen_sql / gen_report 类型的Subagent*
```

markdownlint-cli2 (0.18.1)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)